src/Microsoft.AspNetCore.Mvc.Razor.Host/ViewComponentTagHelperPass.cs

```diff
@@ -187,27 +187,27 @@ private void BuildProcessMethodString(CSharpCodeWriter writer, TagHelperDescript
                 writer.WriteInstanceMethodInvocation(
                     $"(_helper as global::Microsoft.AspNetCore.Mvc.ViewFeatures.IViewContextAware)?",
                     "Contextualize",
-                    new[] { "_context" });
+                    new[] { "ViewContext" });
 
                 var methodParameters = GetMethodParameters(descriptor);
-                var viewContentVariable = "viewContent";
+                var contentVariable = "content";
                 writer.Write("var ")
-                    .WriteStartAssignment(viewContentVariable)
+                    .WriteStartAssignment(contentVariable)
                     .WriteInstanceMethodInvocation($"await _helper", "InvokeAsync", methodParameters);
                 writer.WriteStartAssignment($"{outputVariable}.TagName")
                     .WriteLine("null;");
                 writer.WriteInstanceMethodInvocation(
                     $"{outputVariable}.Content",
                     "SetHtmlContent",
-                    new[] { viewContentVariable });
```

Comment:
Made some changes to make this more compatible with what the old code did. It's just cosmetic.